tst/Pester.RSpec.Configuration.ts.ps1

```diff
@@ -268,6 +268,38 @@ i -PassThru:$PassThru {
             $config.Run.Path = $config.Run.Path.Default
             $config.Run.Path.IsOriginalValue() | Verify-False
         }
+
+        t "TestResult.Enabled is set if TestResult.OutputPath not originalValue even with the Default value" {
+            $config = [PesterConfiguration]::Default
+            $config.TestResult.Enabled.Value | Verify-False
+            $config.TestResult.OutputPath = $config.TestResult.OutputPath.Default
```

Comment:
This seems to be unnecessarily complex. Looking at this test in a month I feel like I will focus on the "default value" being used rather than on what you are actually testing. 

I would rename the test to: TestResult is automatically enabled when OutputPath is set

And below

TestResult is not automatically enabled  when user already set Enabled to $false

Reply:
Making sure that IsOriginalValue works even with the default value, is also nice but imho not the main purpose of this test. And I think we can safely skip it.